packages/webview_flutter/webview_flutter_wkwebview/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Fix typo in a method name, from `setCookieForInsances` to `setCookieForInstances`.
```

Comment:
nit: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Should use `Fixes` instead of `Fix`.